The-New-Fork/openfood
# [Main] add openfood_komodo_node function : getbalance

## Task
[JC-1910](https://thenewfork.atlassian.net/browse/JC-1910)

## Related Task
[JC-1907](https://thenewfork.atlassian.net/browse/JC-1907)

## Summary
add openfood_komodo_node (getbalance function) and unittest

## Changes
- add getbalance function to check balance in one organization (account)
- add unittest for getbalance function

## Others (Optional)
How to run test :
- Go to **test folder** (/deploy-samples/simple-working/organization/blocknotify-python/openfood_lib_dev/tests)
- run command **pytest -s test_komodo_node.py::test_get_balance** (to check result from the test)
- run command **pytest -v test_komodo_node.py::test_get_balance** (to run test and to see that test has been passed)

If you want to run test for all function in test_komodo_node.py, just use : **pytest -v test_komodo_node.py**
